pkg/gds/store/trtl/trtl.go

```diff
@@ -1,15 +1,29 @@
 package trtl
 
 import (
+	"context"
 	"errors"
+	"time"
 
+	"github.com/google/uuid"
 	"github.com/rs/zerolog/log"
 	"github.com/trisacrypto/directory/pkg/gds/client"
 	"github.com/trisacrypto/directory/pkg/gds/models/v1"
 	"github.com/trisacrypto/directory/pkg/gds/store/iterator"
 	"github.com/trisacrypto/directory/pkg/trtl/pb/v1"
+	"github.com/trisacrypto/directory/pkg/utils/wire"
 	gds "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/proto"
+)
+
+// Errors that may occur during Trtl operations.
+var (
+	ErrIncompleteRecord = errors.New("record is missing required fields")
+	ErrEntityNotFound   = errors.New("entity not found")
+	ErrProtocol         = errors.New("unexpected protocol error")
 )
```

Comment:
Should we move these to a `store/errors` package so they can be shared between leveldb and trtl?